### ISSUE TITLE
Nvidia Shield TV 2017

### ISSUE THREAD

**Issue by jandrop** [trakt/Plex-Trakt-Scrobbler#403]
On the Nvidia Shield TV 2017 500Gb this doesn't work, I've tried do the method manual and the method with the WebTools unsuccessfully, I have root access and I checked the permissions and owner and are equals than other pluggins, on the old Nvidia TV 16Gb works fine

**Comment by fuzeman** [comment]
Could you post the latest plugin log file (details [here](https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log-files)) and the output of `cat /proc/cpuinfo` on your device.

-------

Has your "Nvidia TV 16Gb" *(SHIELD 2015?)* been updated to v5.0 (Android Nougat)? this issue might be related to the system upgrade..

**Comment by konecnyna** [comment]
@fuzeman I am having the same issue but I have the `Nvidia Shield TV 16gb 20015 v5.0`. Is there more information related to the issue with that unit?

These are the permissions but the only two plugins that show are 

1. BBC iPlayer
2. Sub-zero subtitles.

<img width="582" alt="screen shot 2017-02-01 at 7 34 25 pm" src="https://cloud.githubusercontent.com/assets/3903695/22532542/acaf3038-e8b5-11e6-932b-354c20378865.png">

**Comment by jandrop** [comment]
@fuzeman My Nvidia Shield 2015 was returned to buy the new one, but i solved the problem yesterday making a symlink from the sdcard directory to Plex plugin directory, now to add a plugin i only have to put on my sdcard and done.

```
cd /sdcard/
mkdir /sdcard/Plug-ins/
ln -sf "/sdcard/Plug-ins/" "/data/data/com.plexapp.mediaserver.smb/Plex Media Server"
```

**Comment by foXaCe** [comment]
none solution for non root shield ?

**Comment by fuzeman** [comment]
@foXaCe 
> none solution for non root shield ?

Nope, root access is currently required to access the `Plug-ins` directory on nvidia shields.

**Comment by foXaCe** [comment]
perhaps .apk addons to contains only trakt.bundle ?
